### ISSUE TITLE
Q: MySQL - access from outside

### ISSUE THREAD

**Issue by JulianVolodia** [pesterhazy/vagrant-lamp-ansible#2]
I want to connect from outside (non-virtual machine) to mysql on that. I won't use phpmyadmin ;) 10.10.10.10 IP address is of course visible for me.

I forwarded port in Vagrantfile (host: 3306, guest: 15306).
I also think the playbook settings fits all. Changed network to Allow All in virtualbox.

Maybe some how created user 'vagrant'@'%' and granted him to everything...

it tells (111) error number, so it's rejected

What should I do?
Thanks for that project! :D

**Comment by pesterhazy** [comment]
Makes sense. I pushed a fix as suggested (you may have to `vagrant destroy` to recreate the mysql users). Can you confirm that it works as intended?

**Comment by JulianVolodia** [comment]
Sorry, I will try it when I have time... but not today :(

For you it works? Have you try it?
